index.js

```diff
@@ -23,6 +23,10 @@ const isObject = value =>
 	!(value instanceof Date);
 
 const camelCaseConvert = (input, options) => {
+	if (typeof input !== 'object') {
```

Comment:
```suggestion
	if (value !== null && typeof input !== 'object') {
```

And can you add `null` to the array in the test you added.